src/routes/+page.svelte

```
@@ -1,2 +1,30 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://svelte.dev/docs/kit">svelte.dev/docs/kit</a> to read the documentation</p>
+<script lang="ts">
+	import { Github } from 'lucide-svelte';
```

ESLint

[error] 'Github' is defined but never used.

(@typescript-eslint/no-unused-vars)